Updated remaining ghostroles for species loadouts

Errant-4 opened the pull request:
## About the PR
Updated all ghostroles (Unknown Shuttle event) and preconfigured spawnable entities (like ERT and Death Squad)  to have the new species-based survival boxes, species-specific EVA tanks where applicable, and vox gear.

Re-enabled vox for Unknown Shuttle spawns

Some NPC loadouts have changed slightly:
EVA-variant ERT starts with a full-sized gas tank in their suitstorage, instead of the double emergency tank they had in their hand. Their gun now spawns in their hands instead of suitStorage.

Death Squad starts with their gun in their Hands instead of their backpack. (the large size of their gun caused issues with spawning their Survival Box from the role loadout once the backpack was already mostly full)

EVA-variant NPCs who are also vox will get a Nitrogen tank in their suit storage, and a second one (that will drop to the ground). Take it for the road.

Fixes #29756

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl: Errant
- fix: Survivors arriving via the Unknown Shuttle event, ERT and CBURN agents, and Death Squad members are now equipped with the appropriate species-specific survival gear.
- tweak: Unknown Shuttle event can once again spawn vox characters.

Errant-4 commented:
Okay never mind, I figured out where apparently I, personally, broke the internals so I'm gonna have to go and move all masks to Loadouts instead of StartingGear

Errant-4 commented:
Internals now properly activate, all done.

IProduceWidgets commented:
Does spawning things in hand work? Last I checked it does not.

IProduceWidgets commented on Resources/Prototypes/Entities/Mobs/Player/human.yml:
```diff
@@ -16,6 +16,8 @@
       hair: HairBald
     - type: Loadout
       prototypes: [SyndicateOperativeGearExtremelyBasic]
+      roleLoadout: [ RoleSurvivalSyndicate ]
+
     - type: RandomMetadata
```
formatting

Errant-4 replied:
The sad part is that I did spot this, I fixed it, then I undid the fix and never fixed it again :/
Thanks

IProduceWidgets commented on Content.Shared/Clothing/LoadoutSystem.cs:
```diff
@@ -120,19 +120,30 @@ private void OnMapInit(EntityUid uid, LoadoutComponent component, MapInitEvent a
-        var loadout = new RoleLoadout(id);
-        loadout.SetDefault(GetProfile(uid), _actors.GetSession(uid), _protoMan, true);
-        _station.EquipRoleLoadout(uid, loadout, proto);
+        // Equip role loadout(s). The component's AddAllRoleLoadouts datafield determines
+        // if all loadouts should be applied, or only a single random one.
+        var idList = new List<ProtoId<RoleLoadoutPrototype>>();
+        if (!component.AddAllRoleLoadouts)
+            idList.Add(_random.Pick(component.RoleLoadout));
```
Won't this mean if you try to give a ghost role a random one from the list of loadouts that the species tank loadouts wont work?

Errant-4 replied:
It does. But I think the random way makes no sense for Loadouts anyway so I don't concern myself with the applications

IProduceWidgets replied:
That's not ideal as they would be really nice for things like visiting shuttles or other somewhat fluff roles.

Errant-4 replied:
Describe to me how you picture this working/being used, because I don't get it. What Loadouts do you want to randomly pick? Can't you just put that in the StartingGear instead, which WILL be a random roll between the ones you select regardless of which way you are using the RoleLoadout list?

IProduceWidgets replied:
> Can't you just put that in the StartingGear instead, which WILL be a random roll between the ones you specified.


Ah, I thought this was what you had changed. That said though, I now don't know why you would need to ever set AddAllRoleLoadouts = true.

Errant-4 replied:
I don't know either, but since that was the original implementation, apparently very intentionally, I left the option in

Errant-4 replied:
> Won't this mean if you try to give a ghost role a random one from the list of loadouts that the species tank loadouts wont work?

Actually, I was wrong. It picks a random RoleLoadout, but each of those can include multiple LoadoutGroups. The survival stuff is in fact in a LoadoutGroup, so it's fine, it can be added. In fact, I am probably going to yank out the entire "include all RoleLoadouts" functionality because I have come to see that it's rather superfluous

Errant-4 commented:
Internals now _also_ properly activate if no Loadouts were added to the StartingGear

Errant-4 commented:
Gonna put this on hold while some things get worked out in #29783 since the PRs are kinda intertwined

Errant-4 commented:
Ok this is now ready to go. Decreased a lot in scope (since the system changes the PR originally started around were already merged on a different PR), it is now essentially just an update to ghostrole/premade humanoid entity prototypes so they all get the species specific survival boxes and gas tanks where applicable.
I have cleaned up the PR and changed the name to reflect this
